www/common/inner/common-modal.js

```diff
@@ -49,7 +49,13 @@ define([
                         // Access modal and the pad is not stored: get the hashes from outer
                         var hashes = priv.hashes || {};
                         // For calendars, individual href is passed via opts
-                        data.href = ((priv.app === 'calendar') && opts.href) || Hash.hashToHref(hashes.editHash || hashes.fileHash, priv.app);
+                        if (priv.app === 'calendar') {
+                            data.href = (priv.app === 'calendar') && opts.href;
```

Comment:
Here you don't need to check if the app is `calendar` in the assignment since it's already checked in the parent `if`. `data.href = opts.href;` is enough